pkg/cache/multi_namespace_cache.go

```diff
@@ -87,6 +94,21 @@ func (c *multiNamespaceCache) GetInformer(ctx context.Context, obj client.Object
 		}
 		informers[ns] = informer
 	}
+
+	isNamespaced, err := objectutil.IsAPINamespaced(obj, c.Scheme, c.RESTMapper)
+	if err != nil {
+		return nil, err
+	}
+
+	if !isNamespaced {
```

Comment:
This is mutually exclusive with getting the namespaced informers, we should either get the namespaced ones or the global ones, not both like we do now

Reply:
ah, yes. right. The obj would either be namespaced or cluster scoped. will make the change.